### PR TITLE
Get ciphers

### LINKED ISSUE #91: listCiphers vs getCiphers
Hey, awesome project!

Is there a reason the name is listCiphers() in this module and not the same as in node.js crypto.getCiphers() ?

Maybe keep both for backwards compatibility, but it's surprising that getHashes is there and getCiphers is not...

### PR DESCRIPTION
**Pull request by calvinmetcalf**
adds getCipher (which was erroneously put in as listCiphers) built on top of #95 to prevent merge conflicts, fixes #91

### CODE REVIEW

**Comment by dcousens** [comment]
ACK

**Comment by dcousens** [comment]
ACK
Fixes #91.

**Comment by calvinmetcalf** [comment]
Ack?

**Comment by dcousens** [comment]
Acknowledge, LGTM

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Dominic Tarr <dominic.tarr@gmail.com> (dominictarr.com)",
   "name": "crypto-browserify",
   "description": "partial implementation of crypto for the browser",
-  "version": "3.8.1",
+  "version": "3.8.3",
```
Comment (dcousens) [inline package.json]: @dominictarr did mention to do this outside of pull requests though.

**Pull request merged**